Testing/WinMLRunnerTest/WinMLRunnerTest.cpp

```diff
@@ -133,6 +180,8 @@ namespace WinMLRunnerTest
             Assert::AreEqual(static_cast<size_t>(2), GetOutputCSVLineCount());
         }
 
+
```

Comment:
Nit: Whitespace

Reply:
Have made correction in the latest commit.